waku/v2/node/config.nim

```diff
@@ -160,6 +160,16 @@ type
       defaultValue: 50000
       name: "store-capacity" }: int
     
+    sqliteOnlyStore* {.
+      desc: "Enable sqlite-only store: true|false",
+      defaultValue: false
+      name: "sqlite-only-store" }: bool
+
+    retentionTime* {.
+      desc: "time the sqlite-only store keeps messages (in seconds)",
+      defaultValue: 30.days.seconds
+      name: "retention-time" }: int64 # TODO: Duration
```

Comment:
We really need sub-commands :). For now, wdyt about having sqlite-related items be prefixed similarly? Perhaps then the conf items could be: `sqlite-retention-time` and `sqlite-store` (not sure, but imo `only` can be dropped - this is a SQLite implementation of store and the description still clarifies that this is different from the memory store with persistence in SQLite).

Reply:
Agreed on both points :). (commit after next call)